pkg/ingress/config_types.go

```diff
@@ -88,6 +89,10 @@ func (t *TargetGroupTuple) validate() error {
 	if t.ServiceName != nil && t.ServicePort == nil {
 		return errors.New("missing servicePort")
 	}
+
+	if aws.Int64Value(t.Weight) < 0 || aws.Int64Value(t.Weight) > 999 {
```

Comment:
Ah, we no longer pull in aws-sdk-go-v2. You need to refer to the Int64Value that is in aws-sdk-go